purchase_order_line_sequence/models/invoice.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+#    Author: Alexandre Fayolle
+#    Copyright 2013 Camptocamp SA
+#    Author: Damien Crier
+#    Copyright 2015 Camptocamp SA
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from openerp import fields, models
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    sequence = fields.Integer(related='purchase_line_id.sequence', store=True)
```

Comment:
The field `sequence` was dropped in v9. IMHO it's better to add a module 'account_invoice_line_sequence' that puts it back. That way we achieve better modularity.